features/admin.users.v1/components/AccountStatusFilterDropdown.tsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { SelectChangeEvent } from "@mui/material/Select";
+
+import Badge from "@oxygen-ui/react/Badge";
+import Box from "@oxygen-ui/react/Box";
+import Checkbox from "@oxygen-ui/react/Checkbox";
+import FormControl from "@oxygen-ui/react/FormControl";
+import ListItemText from "@oxygen-ui/react/ListItemText";
+import MenuItem from "@oxygen-ui/react/MenuItem";
+import Select from "@oxygen-ui/react/Select";
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import React, { FunctionComponent, ReactElement } from "react";
+
+import { USER_ACCOUNT_STATUS_FILTER_OPTIONS } from "../constants/user-management-constants";
+import { AccountStatusFilterOption } from "../models/user";
+
+import "./AccountStatusFilterDropdown.scss";
+
+/**
+ * Props for the AccountStatusFilterDropdown component.
+ */
+interface AccountStatusFilterDropdownProps extends IdentifiableComponentInterface {
+    selectedFilters: string[];
+    onSelectedFiltersChange: (selected: string[]) => void;
+    label?: string;
+}
+
+/**
+ * The AccountStatusFilterDropdown component.
+ *
+ * @param props - Props injected into the component.
+ * @returns React Element.
+ */
+const AccountStatusFilterDropdown: FunctionComponent<AccountStatusFilterDropdownProps> = ({
+    selectedFilters,
+    onSelectedFiltersChange,
```

Comment:
shall we rename this to onChange? since it is called inside accountstatusfilterdropdown, the purpose of onchange is to handle changing selecting filters is implied, IMO.